### DISCUSSION TITLE
“正在获取资源”提示条被删了？

### DISCUSSION THREAD

**Discussion by Ad-closeNN** [Meloong-Git/PCL#5285] [提问] [answered]
来自 Snapshot 2.8.8
![image](https://github.com/user-attachments/assets/bfd5aa9b-95e5-4765-bd1f-c61ba469d89c)

现在啥也不会提示了只会静默等待加载联网主页

**Comment by WForst-Breeze** [comment, marked as answer]
是的

https://github.com/Hex-Dragon/PCL2/commit/2a46c9f5cef710e7e5790b2ac9b41e70b98ea343#diff-d1f4af5c4f4b6119e1f28236026dca01dfdc43ac082df660736a59526677d53fL142
